src/main/java/org/myrobotlab/service/config/Mpu6050Config.java

```diff
@@ -0,0 +1,14 @@
+package org.myrobotlab.service.config;
+
+public class Mpu6050Config extends ServiceConfig {
```

Comment:
I think this config needs the i2c address as well.. but we can do that in a different PR..

Reply:
fixed